bridge_ui/src/components/BridgeWidget/MainActionButton.tsx

```diff
@@ -27,7 +27,6 @@ interface MainActionButtonProps {
 const MainActionButton = ({ onNext }: MainActionButtonProps) => {
   const classes = useStyles()
   const { t } = useTranslation()
-  const { connect: connectAlephium } = useConnect()
```

Comment:
Even though our docs say that we can use `useConnect` to replace the `AlephiumConnectButton`, in reality the `connect` function returned by `useConnect` does not show the modal where the user can select which wallet they want to connect to (WalletConnect / Extension / DW), but instead it chooses the extension wallet. I couldn't connect my mobile wallet.